src/main/java/com/shapesecurity/bandolier/es2017/loader/FileResourceLoader.java

```diff
@@ -0,0 +1,56 @@
+package com.shapesecurity.bandolier.es2017.loader;
+
+import javax.annotation.Nonnull;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+class FileResourceLoader implements IResourceLoader {
+    @Nonnull
+    private final Path root;
+    @Nonnull
+    private final IResourceLoader fallback;
+
+    public FileResourceLoader(@Nonnull Path root) {
+        this(root, EmptyLoader.getInstance());
+    }
+
+    public FileResourceLoader(@Nonnull Path root, @Nonnull IResourceLoader fallback) {
```

Comment:
Rather than adding a `fallback` to every loader, it would maybe be nicer to have a `ChainResourceLoader` or something that would try a list of loaders in order.

Reply:
K, added a `FallbackLoader`.